afl-fuzz.c

```diff
@@ -485,6 +490,26 @@ static void bind_to_free_cpu(void) {
   }
 
   closedir(d);
+#else
+  struct kinfo_proc *procs;
+  size_t nprocs;
+  size_t proccount;
+  cpuset_t c;
+  int s_name[3] = {CTL_KERN, KERN_PROC, KERN_PROC_ALL};
+  if (sysctl(s_name, 3, NULL, &nprocs, NULL, 0) < 0) return;
```

Comment:
Let's not use magic numbers, please add a macro for calculating array length via `sizeof(array) / sizeof(array[0])` and use it instead of literal `3`

Reply:
Sure but in fact I "mimicked" a similar sysctl call above (ie to get number of cpus) I usually try to fit the general style not pb I can change.